users/models.py

```diff
@@ -126,7 +126,7 @@ def has_object_permission(self, obj):
 
     def has_perm(self, perm, obj=None):
         if self.is_staff or self.is_superuser:
-            return super().has_perm(perm, obj)
+            return True
```

Comment:
Suite au partage du bout de code relatif dans Django par @etchegom : https://github.com/django/django/blob/main/django/contrib/auth/models.py#L308 et pour garder le test si l'utilisateur est "actif", est-ce qu'on a pas intérêt a modifier comme suit:

```suggestion
            return self.is_active
```